src/client/src/apps/MainRoute/widgets/spotTile/components/TileSwitch.tsx

```diff
@@ -69,9 +77,7 @@ const TileSwitch: React.FC<Props> = ({
           <TileBooking
             show={!spotTileData.isTradeExecutionInFlight && isRfqStateCanRequest}
             color="blue"
-            onBookingPillClick={() =>
-              rfq.request({ notional: getNumericNotional(notional), currencyPair })
-            }
+            onBookingPillClick={() => rfq.request({ notional, currencyPair })}
```

Comment:
is it worth using a useCallback here?

Reply:
Only if we needed referential identity (e.g. if `TileBooking` was `memo`d), which we don't